WalletWasabi/Wallets/Wallet.cs

```diff
@@ -125,7 +125,7 @@ private set
 
 	public TimeSpan FeeRateMedianTimeFrame => TimeSpan.FromHours(KeyManager.FeeRateMedianTimeFrameHours);
 
-	public bool IsUnderPlebStop => Coins.Confirmed().TotalAmount() <= KeyManager.PlebStopThreshold;
+	public bool IsUnderPlebStop => Coins.Confirmed().TotalAmount() < KeyManager.PlebStopThreshold;
```

Comment:
This is the main change in this PR.
The rest is just updating comments, logs, and renaming a wrong item.